torch/utils/tensorboard/_onnx_graph.py

```diff
@@ -0,0 +1,54 @@
+from tensorboard.compat.proto.graph_pb2 import GraphDef
+from tensorboard.compat.proto.node_def_pb2 import NodeDef
+from tensorboard.compat.proto.versions_pb2 import VersionDef
+from tensorboard.compat.proto.attr_value_pb2 import AttrValue
+from tensorboard.compat.proto.tensor_shape_pb2 import TensorShapeProto
+
+
+def load_onnx_graph(fname):
+    import onnx  # 0.2.1
```

Comment:
Version?

Reply:
@lanpa can you comment here? Thanks.

Reply:
Just tested that onnx 1.5 is working too. The version number should be removed. btw, add a try except here?
![image](https://user-images.githubusercontent.com/2005323/56757480-e97d4f80-67c6-11e9-9b8f-32b07571919d.png)

Reply:
Added a task.